styles.css

```diff
@@ -629,28 +656,62 @@ a {
 @media only screen and (max-width: 768px) {
   .nav-menu {
     position: fixed;
-    left: 100%;
-    top: 5rem;
+    left: 110%;
+    top: 0;
     flex-direction: column;
-    background-color: #fff;
     width: 100%;
-    border-radius: 10px;
-    text-align: center;
+    float: left;
     transition: 0.3s;
     box-shadow: 0 10px 27px rgba(0, 0, 0, 0.05);
+    background-color: #6070ff;
+    margin: 0;
+    border-radius: 0;
+    overflow: visible;
   }
 
   .nav-menu.active {
     left: 0;
+    opacity: 0.96;
+    align-items: flex-start;
+    padding-bottom: 100%;
+    padding-top: 100px;
   }
 
   .nav-item {
-    margin: 40px 0;
+    margin: 20px 0;
+    margin-top: 50px;
+  }
+
+  .nav-link {
+    position: absolute;
+    color: #fff;
+    font-family: Poppins, sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 32px;
+    align-items: flex-start;
+    display: flex;
+    justify-content: space-around;
```

Comment:
- [ ] Kindly make sure that when a user clicks one the mobile menu links(e.g portfolio) it takes them to the appropriate section of the page. 
- [ ] A nav-link is not labeled properly `Home` instead of `Portfolio`.
- [ ] Also the menu should be the full-screen height to match the design.